app/models/workflow.rb

```diff
@@ -48,11 +48,11 @@ def start(data)
 
   private
 
-  def execute_step(step, event)
+  def execute_step(step, event_data)
```

Comment:
notice I changed this from `event` to `event_data`

Reply:
because it isn't an event object..but actually just looked at it. for the first step, it's event data, but for subsequent steps, its actually an event object. So neither is really accurate.